documentation/modules/kafkaconnectors/proc-deploying-kafkaconnector.adoc

```diff
@@ -43,12 +43,12 @@ spec:
 +
 [source,shell,subs="+quotes"]
 ----
-oc apply -f examples/connector/source-connector.yaml
+kubectl apply -f examples/connector/source-connector.yaml
 ----
 
 . Check that the resource was created:
 +
 [source,shell,subs="+quotes"]
 ----
-oc get all --selector strimzi.io/cluster: my-connect-cluster -o name
+kubectl get all --selector strimzi.io/cluster=my-connect-cluster -o name
```

Comment:
Should this be:

```suggestion
kubectl get kctr --selector strimzi.io/cluster=my-connect-cluster -o name
```

`all` will not show the connector resource.

Reply:
As an aside: we could configure our CRs so that we could do `kubectl get strimzi` to show all Strimzi resources, if people think that's useful?

Reply:
I think that would be handy

Reply:
Replaced with `kctr`

Reply:
Thinking about this... I'm not aware that we've documented the shortnames for our CRs.  @laidan6000 can you confirm? We probably should. 

In any case we should be consistent in the docs to always use shortnames or always use long names in order to avoid confusing readers who are not familiar with kube's many ways of spelling resource types.

Reply:
You're correct. We haven't documented the shortnames in full. 

I found a single reference to shortnames in the Kafka topic CRD example:

> "The shortname can be used in CLI commands. For example, `oc get kt` can be used as an abbreviation instead of `oc get kafkatopic`."

Reply:
Thanks @laidan6000